odl/solvers/iterative/statistical.py

```diff
@@ -121,6 +138,131 @@ def mlem(op, x, data, niter=1, noise='poisson', callback=None):
         raise RuntimeError('unknown noise model')
 
 
+def osmlem(op, x, data, niter=1, noise='poisson', callback=None, **kwargs):
```

Comment:
We may also cite section 6.9.4 in [Medical Image Reconstruction: A Conceptual Tutorial by G. Zeng, Springer-Verlag, 2010](http://www.springer.com/gp/book/9783642053689).